lrauv_ignition_plugins/include/lrauv_ignition_plugins/comms/CommsClient.hh

```diff
@@ -25,13 +25,23 @@
 
 #include <functional>
 
+#include <gz/msgs.hh>
 #include <gz/transport/Node.hh>
 
-#include "CommsPacket.hh"
-#include "TopicDefinitions.hh"
+/* #include "CommsPacket.hh" */
 
 #include "lrauv_ignition_plugins/lrauv_acoustic_message.pb.h"
 
+using LRAUVAcousticMessage =
+    lrauv_ignition_plugins::msgs::LRAUVAcousticMessage;
+using MessageType = LRAUVAcousticMessage::MessageType;
+static constexpr auto LRAUVAcousticMessageType =
+    MessageType::LRAUVAcousticMessage_MessageType_Other;
+auto RangeRequest =
+    MessageType::LRAUVAcousticMessage_MessageType_RangeRequest;
+auto RangeResponse =
+    MessageType::LRAUVAcousticMessage_MessageType_RangeResponse;
```

Comment:
@adityapande-1995 why not make these `static constexpr` too?

Reply:
A remnant from ``main``, fixed here : https://github.com/osrf/lrauv/pull/243/commits/feeead6f2803ab83e2944dfde26a5bb02ca96c17